pkg/operator/ceph/disruption/clusterdisruption/osd.go

```diff
@@ -168,130 +182,174 @@ func (r *ReconcileClusterDisruption) reconcilePDBsForOSDs(
 			logger.Debugf("Ceph %q cluster not ready, cannot check Ceph status yet.", request.Namespace)
 			return nil
 		}
-		return errors.Wrapf(err, "could not check cluster health")
+		return errors.Wrapf(err, "failed to check cluster health")
 	}
+
+	activeDrains := len(drainingFailureDomains) > 0
+
+	// create blocking PDBs for all the OSDs that are not being drained right now
+	if activeDrains {
```

Comment:
This is a long method. How about splitting this block out to a new method to handle active drains?

Reply:
using two methods `handleInActiveDrains` and `handleActiveDrains` to split this block.